task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +315,12 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let array = num.toString().split("").map(Number);
```

Comment:
Don't use `Number` to convert string to number)) Use unary `+` or `parseFloat/parseInt`.